<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convertor</class>
 <widget class="QMainWindow" name="Convertor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convertor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Сумма для перевода:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_val"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_conv">
         <property name="text">
          <string>КОНВЕРТИРОВАТЬ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res">
         <property name="text">
          <string>Ответ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_choose">
         <property name="text">
          <string>Выберите номинал:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_from">
             <property name="text">
              <string>Из:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_from">
             <item>
              <property name="text">
               <string>AUD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AZN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GBP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AMD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BYN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BGN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BRL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>HUF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>HKD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DKK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>USD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EUR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>INR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KZT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CAD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KGS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CNY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MDL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NOK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PLN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RON</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XDR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SGD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TJS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TRY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TMT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UZS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UAH</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CZK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SEK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CHF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ZAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KRW</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>JPY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RUB</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_in">
             <property name="text">
              <string>В:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_in">
             <item>
              <property name="text">
               <string>AUD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AZN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GBP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AMD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BYN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BGN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BRL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>HUF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>HKD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DKK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>USD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EUR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>INR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KZT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CAD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KGS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CNY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MDL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NOK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PLN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RON</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XDR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SGD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TJS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TRY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TMT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UZS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UAH</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CZK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SEK</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CHF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ZAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>KRW</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>JPY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RUB</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
